gemviz/aboutdialog.py

```diff
@@ -59,24 +67,25 @@ def doUrl(self, url):
     def doDocsUrl(self):
         """opening documentation URL in default browser"""
         self.setStatus("opening documentation URL in default browser")
-        self.doUrl(__init__.DOCS_URL)
+        self.doUrl(DOCS_URL)
 
     def doIssuesUrl(self):
         """opening issues URL in default browser"""
         self.setStatus("opening issues URL in default browser")
-        self.doUrl(__init__.ISSUES_URL)
+        self.doUrl(ISSUES_URL)
 
     def doLicense(self):
         """show the license"""
-        from licensedialog import LicenseDialog
-        self.close()
+        from .licensedialog import LicenseDialog
+
         self.setStatus("opening License in new window")
-        license=LicenseDialog(self)
+
+        license = LicenseDialog(self)
         license.finished.connect(self.clearStatus)
-        license.open()   
-    
+        license.open()  # modal: must close licensedialog BEFORE aboutdialog
```

Comment:
I liked it better when the `about` dialog closes when I open the `license`; I find it quite annoying to have to close `license` THEN `about`. But that is not a hill I am willing to die on.